From e19f67fe29d49df8b68100f137c1164efa74f732 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Mon, 22 Feb 2016 18:19:44 +0800
Subject: [PATCH 0992/5242] MLK-12432-03: arm: dts: Replace ov5647 mipi sensor
 with ov5640

commit  dcf89f6489df464c0f87e0c624cd8ecee3342b7a from
https://source.codeaurora.org/external/imx/linux-imx.git

ov5647 mipi camera sensor is replaced by ov5640
on imx7D SDB RevB board.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |   11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 93c4e56..f27efac 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -326,7 +326,7 @@
 	status = "okay";
 	port {
 		mipi_sensor_ep: endpoint1 {
-			remote-endpoint = <&ov5647_mipi_ep>;
+			remote-endpoint = <&ov5640_mipi_ep>;
 			data-lanes = <2>;
 			csis-hs-settle = <13>;
 			csis-wclk;
@@ -559,17 +559,18 @@
 		wlf,shared-lrclk;
 	};
 
-	ov5647_mipi: ov5647_mipi@36 {
-		compatible = "ovti,ov5647_mipi";
-		reg = <0x36>;
+	ov5640_mipi: ov5640_mipi@3c {
+		compatible = "ovti,ov5640_mipi";
+		reg = <0x3c>;
 		clocks = <&clks IMX7D_CLK_DUMMY>;
 		clock-names = "csi_mclk";
 		csi_id = <0>;
 		pwn-gpios = <&extended_io 6 GPIO_ACTIVE_HIGH>;
+		AVDD-supply = <&vgen6_reg>;
 		mclk = <24000000>;
 		mclk_source = <0>;
 		port {
-			ov5647_mipi_ep: endpoint {
+			ov5640_mipi_ep: endpoint {
 				remote-endpoint = <&mipi_sensor_ep>;
 			};
 		};
-- 
1.7.9.5

